.travis.yml

```diff
@@ -23,7 +23,7 @@ script:
   - go vet ./...
   - golint ./...
   - make coverage.out
-  - goveralls -coverprofile=coverage.out -service=travis-ci -repotoken $COVERALLS_TOKEN
+  - if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then goveralls -coverprofile=coverage.out -service=travis-ci -repotoken $COVERALLS_TOKEN
```

Comment:
Thanks for the contribution! I think this is missing the closing `; fi` .

Reply:
Indeed...